data/human/rim jobs.txt

```diff
@@ -87,6 +87,12 @@ mission "Han Sizer Month [0]"
 	repeat
 	description `In celebration of Han Sizer month, bring <fare> to all the marked systems and return to <destination> for <payment>.`
 	passengers 2 10 .9
+	substitutions
+		"<on Longjump>" "somewhere nearby."
```

Comment:
These substitutions are grammatically wrong! They won't integrate correctly into the sentence they're being used in.

Reply:
Really? They make:

- As they depart you hear them talking about celebrating their trip with a party somewhere nearby."
- As they depart you hear them talking about celebrating their trip with a party on Longjump."

Those both sound reasonable.

Reply:
If they do resolve to that, that's good. I think they won't, though, and it should be double-checked in game. There might be an extra period at the end, for instance.

I'd this one really is good though, then good

Reply:
Fixed that. Thanks for pointing it out!